concourse/ras-deploy.yml

```diff
@@ -0,0 +1,645 @@
+          echo   \"commit\": \"${commit}\", >> ${output}
+          echo   \"branch\": \"${branch}\", >> ${output}
+          echo   \"name\": \"${repo}\", >> ${output}
+          echo   \"built\": \"${when}\" >> ${output}
+          echo '}' >> ${output}
+          cd ..
+          cp -r django-oauth2-test-source/* django-oauth2-test-source-git-info
+
+  - put: cf-resource-dev
+    params:
+      current_app_name: concourse-django-oauth2-test.dev
+      manifest: django-oauth2-test-source-git-info/manifest.yml
+      path: django-oauth2-test-source-git-info
+      environment_variables:
+        CSRF_ENABLED: "True"
+        DEBUG: "False"
+        DJANGO_SETTINGS_MODULE: proj.settings.cloud_foundry_settings
+        OAUTH2_SUPER_USER: {{dev_oauth2_super_user}}
+        OAUTH2_SUPER_USER_EMAIL: {{dev_oauth2_super_user_email}}
+        OAUTH2_SUPER_USER_PASSWORD: {{dev_oauth2_super_user_password}}
+        SECRET_KEY: {{dev_secret_key}}
+        TESTING: "False"
+        SECURITY_USER_NAME: {{dev_security_user_name}}
+        SECURITY_USER_PASSWORD: {{dev_security_user_password}}
+
+
+- name: ras-integration-tests
+  plan:
+  - get: ras-integration-tests-source
+    trigger: true
+  - get: ras-party-source
+    passed: [ras-party-dev-deploy]
+    trigger: true
+  - get: ras-backstage-source
+    passed: [ras-backstage-dev-deploy]
+    trigger: true
+  - get: ras-frontstage-source
+    passed: [ras-frontstage-dev-deploy]
+    trigger: true
+  - get: ras-api-gateway-source
+    passed: [ras-api-gateway-dev-deploy]
+    trigger: true
+  - get: ras-backstage-ui-source
+    passed: [ras-backstage-ui-dev-deploy]
+    trigger: true
+  - get: django-oauth2-test-source
+    passed: [django-oauth2-test-dev-deploy]
+    trigger: true
+  - get: ras-secure-message-source
+    passed: [ras-secure-message-dev-deploy]
+    trigger: true
+  - get: ras-collection-instrument-source
+    passed: [ras-collection-instrument-dev-deploy]
+    trigger: true
+  - task: build_test
+    config:
+      platform: linux
+      image_resource:
+        type: docker-image
+        source:
+          repository: python
+          tag: "latest"
+      inputs:
+        - name: ras-integration-tests-source
+      run:
+        path: sh
+        args:
+        - -exc
+        - |
+          echo "Passed"
+
+- name: ras-party-test-deploy
+  plan:
+  - get: ras-party-source
+    passed: [ras-integration-tests]
+    trigger: true
+  - put: cf-resource-test
+    params:
+      current_app_name: concourse-ras-party.test
+      manifest: ras-party-source/manifest.yml
+      path: ras-party-source
+      environment_variables:
+        SECURITY_USER_NAME: {{test_security_user_name}}
+        SECURITY_USER_PASSWORD: {{test_security_user_password}}
+        case-service.host: 'casesvc-test.apps.devtest.onsclofo.uk'
+        case-service.port: '80'
+        collectionexercise-service.host: 'collectionexercisesvc-test.apps.devtest.onsclofo.uk'
```

Comment:
Do all of the services all need all of these env vars?

Reply:
I think so? I've just copied from the jenkins job but it would be good to cut them down to the bare minimum if they aren't needed